### ISSUE TITLE
Feature Request: Can dashed pie charts be added, here is an example. (PieChartPro)

### ISSUE THREAD

**Issue by AyoCodess** [Abhinandan-Kushwaha/react-native-gifted-charts#584]
![Screenshot 2024-03-26 at 12 02 26](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/72066389/e79d66b7-069b-4c53-a163-7713a9e127bd)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @AyoCodess 👋
Thanks for requesting this feature.
It is now available from version `1.4.9` onwards.

It can be achieved using the following props-

1. ring
2. strokeWidth
3. strokeColor
4. strokeDashArray

Here's an example-
```js
const ringData = [
  {value: 5, strokeColor: '#F5C734'},
  {value: 15, strokeColor: '#EEE2D2'},
];

return (
  <PieChartPro
    data={ringData}
    ring
    strokeWidth={20}
    strokeDashArray={[4, 15]}
    pieInnerComponent={() => {
      return <Text style={{fontSize: 80, fontWeight: '100'}}>25%</Text>;
    }}
  />
);
```
The output of the above code is-

<img width="286" alt="ring" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/20596944/d171bdca-688b-48a3-93be-87caf43d1409">

Please use the latest version of the library.

**Comment by AyoCodess** [comment]
Awesome!
